packages/config/src/android/Package.ts

```diff
@@ -25,14 +25,16 @@ function getMainApplicationPath({
     cwd: packageRoot,
   });
   // If there's more than one, we'll probably have a problem.
-  return mainApplications[0];
+  // Also, glob always returns a posix formatted path (even on windows),
+  // lets normalize that so we can use it with `.split(path.sep)`
+  return path.normalize(mainApplications[0]);
```

Comment:
🔥

Reply:
@byCedric - maybe we should make a wrapper around glob that always returns a normalized path to stop this from biting us in the future